apps/styleguide/src/layout/Molgenis.vue

```diff
@@ -80,6 +83,31 @@ export default {
     };
   },
   computed: {
+    crumbs() {
+      this.$route;
+      let path = decodeURI(window.location.pathname).split("/");
+      let url = "/";
+      let result = { molgenis: url };
+      if (window.location.pathname != "/apps/central/") {
+        path.forEach((el) => {
+          if (el != "") {
+            url += el + "/";
+            result[el] = url;
+          }
+        });
+      }
+      if (this.$route) {
+        path = decodeURI(location.hash).substr(1).split("/");
+        url += "#";
+        path.forEach((el) => {
+          if (el != "") {
+            url += "/" + el;
+            result[el] = url;
+          }
+        });
+      }
+      return result;
+    },
```

Comment:
maybe tell view controle the route via slot  or props instead of generating

Reply:
ah, how could that work?

Reply:
View X includes a breadcrumb component in its template and passes the crumbs as props

Reply:
I have been thinking something along those lines, but thought: lets try to not do that and see if we can succeed

Reply:
By generating the breadcrumbs from the url it forces a 1 to 1 mapping of the data storage structure to the content hierarchy / global nav. This removes the ability to have a task focused application layout.

Reply:
That is exactly what I want, I think we have responsibility to create transparancy of the data to our users. This is a scientific app. There are certainly ways to bypass this, either using query variables or artificial params. However, I want developers to think 3 times before they start hiding essential information from our users. Because I repeat myself: this is a scientific application that aims to improve FAIRness of data. Then it is essential that we don't hide the data.